Sample App/Custom Widgets/CustomWidgetTimer.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  CustomWidgetTimer.swift
+//  Sample App
+//
+//  Created by Jelzon Monzon on 1/5/21.
+//  Copyright © 2021 LiveLike. All rights reserved.
+//
+
+import UIKit
+
+class CustomWidgetBarTimer: UIView {
+
+    var progressView: UIView = {
+        let view = UIView()
+        view.backgroundColor = UIColor(red: 250/255, green: 200/255, blue: 60/255, alpha: 1)
```

Comment:
btw this shows up black for me, `UIColor(red: 250, green: 200, blue: 60, alpha: 1)` works

Reply:
Thats because we have this extension on the TestApp

```extension UIColor {
    convenience init(red: Int, green: Int, blue: Int, alpha: CGFloat = 1.0) {
        let newRed = CGFloat(red) / 255

        let newGreen = CGFloat(green) / 255

        let newBlue = CGFloat(blue) / 255

        self.init(red: newRed, green: newGreen, blue: newBlue, alpha: alpha)
    }
}```

We should make sure to use the native init for sample app

Reply:
Good catch